docs/dev/framework/routing/legacy-parameters.md

```diff
@@ -0,0 +1,123 @@
+---
+title: "Legacy Parameters"
+description: "Legacy routing parameters in Contao"
+alias:
+  - /framework/routing/legacy-parameters/
+---
+
+
```

Comment:
I'm missing like a general warning that clearly states that if you build things from scratch, you should never use `auto_item` anymore but instead the page type :)